crates/sui/tests/data/linter/sources/suppression_stats.move

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Is there an `exp` for this file?

Reply:
There isn't. This is tested as an CLI test (in the sui crate) checking if the output contains the right type of stats. I added a comment explaning this.